substratevm/JFR.md

```diff
@@ -0,0 +1,79 @@
+This page covers how to use JDK Flight Recorder (JFR) with Java native images
```

Comment:
Just want to confirm here, are we allowed to use the Java Flight Recorder name here since it's an Oracle project? I recall part of the open sourcing process for JFR/JMC was the open source version being named/referenced as JDK Flight Recorder and JDK Mission Control (instead of Java).

Reply:
@christianwimmer can you answer this question?

Reply:
The wiki page says "JDK Flight Recorder". In history section you see _"In 2018 Java Flight Recorder was open sourced and released as part of OpenJDK 11.[4] When open sourced it was rebranded JDK Flight Recorder, due to Java trademark issues."_